Classes/Repository/RepositorySummarySectionController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  RepositorySummarySectionController.swift
+//  Freetime
+//
+//  Created by Sherlock, James on 29/07/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import IGListKit
+
+final class RepositorySummarySectionController: ListGenericSectionController<IssueSummaryModel> {
+    
+    private let client: GithubClient
+    private let repo: RepositoryLoadable
+    
+    init(client: GithubClient, repo: RepositoryLoadable) {
+        self.client = client
+        self.repo = repo
+        super.init()
+    }
+    
+    override func sizeForItem(at index: Int) -> CGSize {
+        guard let width = collectionContext?.containerSize.width else { fatalError("Missing context") }
```

Comment:
Off topic and just more of a question for @rnystrom. After I have been using IGListKit for a while I have come to the conclusion that most of the time, what I want here is not the raw size of the container, but the one with the insets `collectionContext?.insetContainerSize`. Thus if there are higher level insets on this collectionView, I do not have to care about them and still size my items accordingly :) Is there a reason (ryan) you are not defaulting to that?